subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv_iso.c

```diff
@@ -625,11 +625,27 @@ static void isr_tx_common(void *param,
 
 	/* Get ISO data PDU, not control subevent */
 	if (!pdu) {
-		uint8_t payload_index;
+		uint16_t payload_index;
 
-		payload_index = (lll->bn_curr - 1U) +
-				(lll->ptc_curr * lll->pto);
-		payload_count = lll->payload_count + payload_index - lll->bn;
+		if (lll->ptc_curr) {
+			uint8_t ptx_idx = lll->ptc_curr - 1; /* max. nse 5 bits */
```

Comment:
Since `ptc_curr` is `uint32_t ptc_curr:4;`, then shouldn't this be 
```suggestion
			uint8_t ptx_idx = lll->ptc_curr - 1; /* max. nse 4 bits */
```

Which then also affect the bits of several of the values below?

Reply:
> Now I do not remember why `ptc` is 4 bits! ... it can be as much as (NSE - 1); 1 transmission followed by (NSE - 1) pre-transmissions.
> 
> @Tronil and @wopu-ot , are you testing the boundary conditions internally in your tests for this PR changes?

We do not have extensive test coverage on the broadcaster side, no

Reply:
@Tronil while testing this PR for PTO=2 and Group Count = 2, I discovered a bug in ull_sync_iso.c that subsequent BIS subevent reception failed; `ptc` was not assigned the correct value. This bug will not show up if only one BIS is synchronized to.

Reply:
Shouldn't the comment be updated?